### ISSUE TITLE
focus

### ISSUE THREAD

**Issue by TomasLongo** [codex-team/editor.js#758]
Hi,

is there a way to detect if the editor is in focus or blurred?

My use case is that i have multiple editors in my site and I want to display/hide certain control elements depending on if the editor is in focus. 

Thanks,
Tomas

**Comment by CavalcanteLeo** [comment]
I wanna know this too

**Comment by hata6502** [comment]
https://github.com/codex-team/editor.js/pull/1224

**Comment by arnaud-zad** [comment]
Yes it makes sense to have this feature. Looking for focus event is a common feature. 

One use case would be to applied a css class to another HTML element when the editor is focused. 

The PR of @hata6502 looks great, thank you. It's sad it wasn't accepted.
